setup/azurecmdfiles.wxi

```diff
@@ -331,6 +336,42 @@
                             </Component>
                         </Directory>
                     </Directory>
+                    <Directory Id="dirDD31E938D66F302AFB1DF0459D461C42" Name="AzureRM.Compute.ManagedService">
```

Comment:
This should not be in the installer - none of the preview packages should be in the installer

Reply:
Do we have a build configuration target for non-preview modules?

Reply:
I've removed the preview modules by hand from the wxi file.